engine/tests/classes/Elgg/Mocks/Database/Plugins.php

```diff
@@ -73,14 +73,19 @@ public function find($status = 'active') {
 		return $this->_plugins;
 	}
 
+	public function generateEntities() {
+		$this->addTestingPlugin(ElggPlugin::fromId('test_plugin', ''));
```

Comment:
i think we also need to trigger parent::generateEntities

Reply:
@dploeger can you add that?

Reply:
Absolutely